index.d.ts

```diff
@@ -0,0 +1,263 @@
+      offset: number,
+      limit: number,
+    }): Promise<object>;
+    getMoveDamageClassByName(name: string | number | string[] | number[], dict?: {
+      offset: number,
+      limit: number,
+    }): Promise<object>;
+    getMoveLearnMethodByName(name: string | number | string[] | number[], dict?: {
+      offset: number,
+      limit: number,
+    }): Promise<object>;
+    getMoveTargetByName(name: string | number | string[] | number[], dict?: {
+      offset: number,
+      limit: number,
+    }): Promise<object>;
+    getLocationByName(name: string | number | string[] | number[], dict?: {
+      offset: number,
+      limit: number,
+    }): Promise<object>;
+    getLocationAreaByName(name: string | number | string[] | number[], dict?: {
+      offset: number,
+      limit: number,
+    }): Promise<object>;
+    getPalParkAreaByName(name: string | number | string[] | number[], dict?: {
+      offset: number,
+      limit: number,
+    }): Promise<object>;
+    getRegionByName(name: string | number | string[] | number[], dict?: {
+      offset: number,
+      limit: number,
+    }): Promise<object>;
+    getAbilityByName(name: string | number | string[] | number[], dict?: {
+      offset: number,
+      limit: number,
+    }): Promise<object>;
+    getCharacteristicById(id: number | number[], dict?: {
+      offset: number,
+      limit: number,
+    }): Promise<object>;
+    getEggGroupByName(name: string | number | string[] | number[], dict?: {
+      offset: number,
+      limit: number,
+    }): Promise<object>;
+    getGenderByName(name: string | number | string[] | number[], dict?: {
+      offset: number,
+      limit: number,
+    }): Promise<object>;
+    getGrowthRateByName(name: string | number | string[] | number[], dict?: {
+      offset: number,
+      limit: number,
+    }): Promise<object>;
+    getNatureByName(name: string | number | string[] | number[], dict?: {
+      offset: number,
+      limit: number,
+    }): Promise<object>;
+    getPokeathlonStatByName(name: string | number | string[] | number[], dict?: {
+      offset: number,
+      limit: number,
+    }): Promise<object>;
+    getPokemonByName(name: string | number | string[] | number[], dict?: {
+      offset: number,
+      limit: number,
+    }): Promise<object>;
+    getPokemonEncounterAreasByName(name: string | number | string[] | number[], dict?: {
+      offset: number,
+      limit: number,
+    }): Promise<object>;
+    getPokemonColorByName(name: string | number | string[] | number[], dict?: {
+      offset: number,
+      limit: number,
+    }): Promise<object>;
+    getPokemonFormByName(name: string | number | string[] | number[], dict?: {
+      offset: number,
+      limit: number,
+    }): Promise<object>;
+    getPokemonHabitatByName(name: string | number | string[] | number[], dict?: {
+      offset: number,
+      limit: number,
+    }): Promise<object>;
+    getPokemonShapeByName(name: string | number | string[] | number[], dict?: {
```

Comment:
Hi, sorry, all these methods (`get*byName`, `get*byId`) don't support the options dict.

Reply:
No problem, done.